userguide/jails.rst

```diff
@@ -52,15 +52,14 @@ intended jails and plugins. The
 screen displays a message and button to :guilabel:`CREATE POOL` if no
 pools exist on the %brand% system.
 
-Multiple pools can be activated to store iocage jails and plugins. After
-a pool is created, the
-:menuselection:`Jails`
-page displays an :guilabel:`Activated Pool` section. This shows which
-pool and iocage dataset is active with %brand%. Click :guilabel:`CONFIG`
-to view the option to choose another pool or dataset to activate with
-iocage. :guilabel:`ACTIVATE` another pool to refresh the
-:menuselection:`Jails`
-list with any jails that exist on the chosen pool or dataset.
+If a pool exist, but no pool has been activated with iocage, a
+drop-down appears to select a pool. Select an appropriate pool and
+click :guilabel:`CHOOSE`.
+
+Multiple pools can be activated to store iocage jails and plugins. To
```

Comment:
Not clear what this means.  It sounds like I can choose more than one pool for jails and plugins, but that's not what the UI does.  It just lets you choose a different one.  It's not necessary to refer to iocage here at all, another implementation detail.